BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeError.kt

```diff
@@ -87,7 +88,12 @@ open class BraintreeError : Parcelable {
             return errors
         }
 
-        internal fun fromGraphQLJsonArray(graphQLErrors: JSONArray?): List<BraintreeError> {
+        /**
+         * @suppress
+         */
+        @JvmStatic
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun fromGraphQLJsonArray(graphQLErrors: JSONArray?): List<BraintreeError> {
```

Comment:
This is `protected` in Java, so opening up its scope to adhere to semantic versioning. Companion objects cannot have protected methods.